tool/pftool.cc

```diff
@@ -717,6 +726,95 @@ static void daq( const std::string& cmd, PolarfireTarget* pft ) {
       fclose(f);
     }
   }
+  if (cmd=="EXTERNAL") {
```

Comment:
This command is designed for using your dmaRead server - are you still operating using this mode? If you are that's fine, we will keep it, but if you aren't we should remove it or update it to use the `daq_dma_X` functions now in RogueWishboneInterface.

Reply:
Resolved by 5b1bb35c8de6d902fc31c1df879b21e43db5c24a